speakers.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: noheader
```

Comment:
Inte helt uppenbart, men `title:` behövs för att det ska bli rätt i `<title>` också. När jag fixat till våra layouts så kommer sidans `<h1>` populeras med detta värde.
```suggestion
layout: noheader
title: Speakers
```